src/Map/ProspectorOverlayLayer.cs

```diff
@@ -444,11 +366,6 @@ public override void OnMapOpenedClient()
 
         public void RebuildMap(bool rebuildTexture = false)
         {
-            foreach (var component in _components)
```

Comment:
any reason why the map components don't get disposed anymore ?

Reply:
IIRC, `IDisposable` is used to explicitly free resources such as loaded textures. However, the components (`ProspectorOverlayMapComponent`) themselves do not have any resources that need to be freed, i.e., they don't override the `Dispose` method. The textures are instead managed within the `ProspectorOverlayLayer` class. So this cleanup code here just called the empty implementation of `Dispose` in the `MapComponent` class.